modules/rules_java/metadata.json

```diff
@@ -25,7 +25,8 @@
         "7.0.6",
         "7.1.0",
         "7.2.0",
-        "7.3.0"
+        "7.3.0",
+        "6.5.2"
```

Comment:
Please keep this sorted manually.

@kormide Does the Publish to BCR app support sorting this automatically?

Reply:
It appears not but that would be an easy fix. I'll make an issue.

Reply:
https://github.com/bazel-contrib/publish-to-bcr/issues/89

Reply:
@fmeum I've updated the list manually here. Ready for approval / merge.
fyi @hvadehra